app/controllers/iiif/auth/v2/probe_service_controller.rb

```diff
@@ -59,8 +60,22 @@ def auth_probe_result(file, cocina)
             else
               AuthProbeResult2.ok
             end
-          else
+          elsif file.stanford_restricted?
             AuthProbeResult2.unauthorized(unauthorized_heading(file))
+          else
+            AuthProbeResult2.forbidden(*forbidden_fields(file))
+          end
+        end
+        # rubocop:enable Metrics/AbcSize, Metrics/PerceivedComplexity
+
+        def forbidden_fields(file)
+          if file.embargoed?
+            [I18n.t('probe_service.embargoed', date: file.embargo_release_date.to_date), I18n.t('probe_service.embargoed_icon')]
+          elsif file.restricted_by_location?
+            [I18n.t('probe_service.location', location: Settings.user.locations.labels.send(file.location)),
+             I18n.t('probe_service.location_icon')]
+          elsif file.no_download?
```

Comment:
Do we need a conditional here?  Isn't this the only possible outcome?

Reply:
I was considering that but the original code didn't have an else on headings. I don't think we necessarily want to just do else for everything https://github.com/sul-dlss/stacks/blob/760c866ee1065a12ce7746121446029f3911803b/app/controllers/iiif/auth/v2/probe_service_controller.rb#L68-L78.

Reply:
The other option is that it returns `nil` here, and we aren't testing or expecting that. I'd rather have deterministic behavior (even if we have to fix it later).

Reply:
updated